prow/scripts/cluster-integration/kyma-gke-long-lasting.sh

```diff
@@ -197,11 +196,27 @@ function generateAndExportLetsEncryptCert() {
         --server https://acme-v02.api.letsencrypt.org/directory \
         --dns-google-propagation-seconds=600 \
         -d "*.${DOMAIN}"
-
+	
     TLS_CERT=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/fullchain.pem | tr -d '\n')
     export TLS_CERT
     TLS_KEY=$(base64 -i ./letsencrypt/live/"${DOMAIN}"/privkey.pem   | tr -d '\n')
     export TLS_KEY
+	#encrypt the tls cert
+	gcloud kms encrypt --location global \
+	--keyring kyma-prow \
+	--key projects/kyma-project/locations/global/keyRings/kyma-prow/cryptoKeys/kyma-prow-encryption\
+	--plaintext-file ./letsencrypt/live/"${DOMAIN}"/fullchain.pem  \
+	--ciphertext-file "nightly-aks-tls-integration-app-client-cert.encrypted"
```

Comment:
you edit the script for provisioning GKE, but the encrypted file contains "AKS". Please move this logic to helper script and use it for script provisioning long-lasting clusters on GKE and AKS.

Reply:
oops, https://status.build.kyma-project.io/?job=kyma-aks-nightly. I did not realize how much the aks fails as well. Thoughts for a name for the certificate? nightly-cloud-tls-integration-app-client-cert.encrypted?